pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -9,12 +9,19 @@ const (
 	SecurityContextConstraintPrivileged = "privileged"
 	// SecurityContextConstraintRestricted is used as the name for the system default restricted scc.
 	SecurityContextConstraintRestricted = "restricted"
+	// SecurityContextConstraintNonRoot is used as the name for the system default non-root scc.
+	SecurityContextConstraintNonRoot = "nonroot"
+	// SecurityContextConstraintHostMount is used as the name for the system default host mount only scc.
+	SecurityContextConstraintHostMount = "hostmount"
```

Comment:
That user can just create an SCC.  Nothing special about these.  Also, why does that user need host mount?  The PV shouldn't need to go through the SCC.